src/views/Farms/components/SearchInput.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useRef } from 'react'
+import { Input } from '@pancakeswap-libs/uikit'
+import styled from 'styled-components'
+
+const StyledInput = styled(Input)`
+  border-radius: 16px;
+  transition: 0.3s;
```

Comment:
What is this transition for? Try and avoid setting transition for every property.